framework/contracts/account/manager/tests/upgrades.rs

```diff
@@ -911,6 +912,26 @@ fn install_app_with_proxy_action() -> AResult {
     abstract_integration_tests::manager::install_app_with_proxy_action(chain)
 }
 
+#[test]
+fn native_not_migratable() -> AResult {
```

Comment:
```suggestion
fn ibc_client_not_migratable() -> AResult {
```

Reply:
The reason why it's not migratable is because module type native is not migratable. We just have only 1 installable native module right now :)